### PR TITLE
Azure Static Web Apps content

### PR DESCRIPTION
**Pull request by aaronamm**
Add Azure Static Web Apps content

### CODE REVIEW

File: content/cloud-hosting/azure/azure-static-web-apps.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    --token <GITHUB_PERSONAL_ACCESS_TOKEN>
+  ```
+- For RESOURCE_GROUP_NAME, use `az group list --output table`.
+- For LOCATION_NAME, use `az account list-locations --output table`.
+- For GITHUB_REPOSITORY_URL_OF_STATIC_SITE, use `https://github.com/account-name/repository-name` pattern.
+- For REPOSITORY_BRANCH_TO_DEPLOY, use a branch that you want to deploy source code to a static app, e.g. main, develop.
+- For GITHUB_PERSONAL_ACCESS_TOKEN, please refer to
+  ["Creating a personal access token"](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) document.
+  Get only repo (full control of private repo) permission.
+- Learn more about `az static webapp create` command, please refer to
```
Comment (dmakeroam) [inline content/cloud-hosting/azure/azure-static-web-apps.md]: typo? az staticwebapp?

**Review by dmakeroam — COMMENTED** [review]
LGTM

File: content/cloud-hosting/azure/azure-static-web-apps.md (hunk 1 above)
Comment (aaronamm) [reply content/cloud-hosting/azure/azure-static-web-apps.md]: @dmakeroam  Good catch. 
Thanks.

**Review by dmakeroam — APPROVED** [review]

File: content/cloud-hosting/azure/azure-static-web-apps.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    --token <GITHUB_PERSONAL_ACCESS_TOKEN>
+  ```
+- For RESOURCE_GROUP_NAME, use `az group list --output table`.
+- For LOCATION_NAME, use `az account list-locations --output table`.
+- For GITHUB_REPOSITORY_URL_OF_STATIC_SITE, use `https://github.com/account-name/repository-name` pattern.
+- For REPOSITORY_BRANCH_TO_DEPLOY, use a branch that you want to deploy source code to a static app, e.g. main, develop.
+- For GITHUB_PERSONAL_ACCESS_TOKEN, please refer to
+  ["Creating a personal access token"](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) document.
+  Get only repo (full control of private repo) permission.
+- Learn more about `az staticwebapp create` command, please refer to
```
Comment (aaronamm) [inline content/cloud-hosting/azure/azure-static-web-apps.md:36]: @dmakeroam Since it is very minor update, I forced push to this PR.
Thanks.

**Pull request merged**